lib/susedistribution.pm

```diff
@@ -276,6 +276,7 @@ sub ensure_installed {
         die "pkcon install did not succeed, return code: $ret";
     }
     send_key("alt-f4");    # close xterm
+    assert_screen('generic-desktop');
```

Comment:
if you want to do that please do it as a separate PR to make that change more obvious as it will have a big impact

Reply:
That's funny, I thought you would complain that I didn't wanted to do it on another PR blocking my current PR and ticket, and then I decided to include it on this PR.

But I have put it on a different commit so it can be reverted. Do you really just want another PR or another ticket?